lib/pharos/host/el7/el7.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Host
+    class El7 < Configurer
```

Comment:
This should also implement `upgrade_kubeadm`, not upgrade path fails with:
```
/Users/jussi/code/pharos-cluster/lib/pharos/host/configurer.rb:119:in `abstract_method!': This is an abstract base method. Implement in your subclass. (NotImplementedError)
	from /Users/jussi/code/pharos-cluster/lib/pharos/host/configurer.rb:43:in `upgrade_kubeadm'
	from /Users/jussi/code/pharos-cluster/lib/pharos/phases/upgrade_master.rb:30:in `upgrade_kubeadm'
	from /Users/jussi/code/pharos-cluster/lib/pharos/phases/upgrade_master.rb:22:in `call'
...
```

Reply:
Yes, currently trying to find a way to upgrade just `kubeadm` with rpm...

Reply:
Fixed.